templates/app/env

```diff
@@ -4,5 +4,6 @@ FOREST_ENV_SECRET=<%= forestEnvSecret %>
 <% if (!ssl) {%>#<% } %>SSL_DATABASE=true
 <% if (!encrypt) {%>#<% } %>ENCRYPT_DATABASE=true
 <% if (forestUrl) {%>FOREST_URL=<%= forestUrl %><% } %>
+<% if (devRenderingId) {%>DEV_RENDERING_ID=<%= devRenderingId %><% } %>
```

Comment:
`DEV_RENDERING_ID` ? It's not really explicit what is its purpose, `REDIRECT_DEFAULT_RENDERING_ID` ?

Reply:
I don't agree. The variable is really "the development rendering id" and the use case today is for redirection but it could be used later for another purpose.